### DISCUSSION TITLE
Adding quotes to the diagram

### DISCUSSION THREAD

**Discussion by PatWekan** [nickpadd/EuropeanFootballLeaguePredictor#72] [Ideas]
Hello,

from my point of view it would be helpful to see the quotes/converted quotes additionally to the probabilites in the spider diagram

Just as an idea to share here :)

Pat

**Comment by nickpadd** [comment]
Can you please explain what do you mean quotes/converted quotes?
It might be possible to enhance the output figures.

**Comment by PatWekan** [comment]
Hello

with pleasure: by now you put e.g. prob. of 33% on the diagramm. It would be great to see the quote 1/0.33 = 3.03 addiotionally :)

**Reply by nickpadd** [reply]
If I understand correctly you mean the implied odds of the probabilites?

The reason I give probability is because I thought it would be the best indicator for what bet my model thinks of as more propable to happen than the odds provider.

It would definetely be something that can be added in a next update. Maybe when hovering over the exact bet in the spider diagram I can add an 'Implied Odds' row.

**Comment by PatWekan** [comment]
Hi

thanks for your great support and i am happy to use your model

Many thanks for
